### DISCUSSION TITLE
Translation for es-ES on PR attach files button is wrong

### DISCUSSION THREAD

**Discussion by asieraduriz** [community/community#19983] [Projects and Issues]
![image](https://user-images.githubusercontent.com/4102787/177180456-bd7e5204-bfc0-41d8-93a8-605d415dd6c6.png)
Given I'm a user/PR author
When I scroll down to add my FIRST attachment
Then I see an incorrect es-ES copy

Where it says `Ninguno archivo selec.` it should say `Ningún archivo selec.`

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
